src/dex-ui-components/base/Icons/GlobeIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Color } from "../../themes";
+import { createIcon } from "@chakra-ui/react";
+
+export const GlobeIcon = createIcon({
+  displayName: "GlobeIcon",
+  viewBox: "0 0 20 20",
+  path: (
+    <path
+      d="M18.333 10A8.333 8.333 0 0 1 10 18.333M18.333 10A8.333 8.333 0 0 0 10 
```

Comment:
You can use https://jakearchibald.github.io/svgomg/ to reduce the size of the SVG elements (mostly the `d` attribute on the path `element`). You may have already done this but I can't determine if the path is conpressed by only reading the coordinates.

Reply:
I have already compressed the icons with the above.